src/test-utils.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import './globals.d';
```

Comment:
```suggestion
import './globals.js';
```

I'm not 100% sure about this

Reply:
It's definition file, with `declare module`, so it cannot be compiled to `.js` file. Compiler complains about that.

```tsx
⋊> ~/P/c/ckeditor5-integrations-common on ck/fix-extensions ⨯ yarn run build                              09:46:16
yarn run v1.22.19
$ vite build
vite v5.4.1 building for production...
✓ 2 modules transformed.
x Build failed in 2.70s
error during build:
Could not resolve "./globals.js" from "src/index.ts"
file: /home/mateusz/Projekty/cksource/ckeditor5-integrations-common/src/index.ts
```